src/HydraAuction/Tx/Escrow.hs

```diff
@@ -145,8 +140,8 @@ announceAuction terms = do
         , validityBound = (Nothing, Just $ biddingStart terms)
         }
 
-startBidding :: AuctionTerms -> Runner ()
-startBidding terms = do
+startBidding :: AuctionTerms -> ApprovedBidders -> Runner ()
```

Comment:
I'll add that, tbh I realised I wasn't 100% sure why we have it. The spec says it is to avoid with tampering, but when would that happen?
We already enforce that the standing bid does not change thelist of approved bidders in its validator

Reply:
Hm, you are probably right. AFAI remember `ApprovedBiddersHash` was in old spec, and list was added later. So I guess it is just outdated.

I propose not to implement it now and wait for @GeorgeFlerovsky return to ask  if we missed something.

Reply:
Perfect, I have added that anyway. I can keep it in a separate PR, and when @GeorgeFlerovsky is back we can disciuss